modules/client-configuration/pages/cobbler.adoc

```diff
@@ -184,15 +181,12 @@ zypper install susemanager-tftpsync-recv
 
 
 .Procedure: Configuring TFTP on a Proxy
-
 . On the {productname} Proxy, as root, run the [command]``configure-tftpsync.sh`` script.
-. The script interactively asks you for details on the host names and IP addresses of the {productname} Server and Proxy, as well for the location of the [path]``tftpboot`` directory on the Proxy.
+. The script requires the host names and IP addresses of the {productname} Server and Proxy, as well the location of the [path]``tftpboot`` directory on the Proxy.
```

Comment:
I am not an expert on this topic, but looking at the script it seems only requires Proxies IP's (not Server).
This makes sense because you are calling the script from the server.
The script includes the following help:
```
print_help () {
        cat <<HELP
usage: configure-tftpsync.sh proxy1.domain.top proxy2.domain.top ...

HELP
exit 1
}
```

Reply:
Updated. How does one call that help text, if not with `configure-tftpsync.sh --help` ?

Reply:
Likely the author of the script forgot to support the `--help` option.
It is trivial to do it, so maybe we could just support it as in other scripts.

Reply:
Ah, that's annoying. What should the docs say in this case?

Reply:
This option is supported.